packages/shared/src/lib/featureManagement.ts

```diff
@@ -48,6 +74,7 @@ const feature = {
     'experience_level',
     ExperienceLevelExperiment.Control,
   ),
+  featureTheme: new Feature('feature_theme', eTheme),
```

Comment:
Default value should be `{}`.

Reply:
https://app.growthbook.io/features/feature_theme growthbook created 

default value: {}
for team it is set to the star wars config